tests/rados/test_osd_ecpool_inconsistency_scenario.py

```diff
@@ -50,29 +50,45 @@ def run(ceph_cluster, **kw):
     objectstore_obj = objectstoreToolWorkflows(node=cephadm)
     mon_obj = MonConfigMethods(rados_obj=rados_obj)
     client_node = ceph_cluster.get_nodes(role="client")[0]
-    scrub_obj = RadosScrubber(node=cephadm)
-    obj_count = ""
+    wait_time = 45
 
     try:
         # set global autoscaler to off
         rados_obj.configure_pg_autoscaler(**{"default_mode": "off"})
         # Creating ec pool
         ec_config = config.get("ec_pool")
         pool_name = ec_config["pool_name"]
-
         req_no_of_objects = config.get("inconsistent_obj_count")
 
+        # Set the default values
+        mon_obj.remove_config(section="osd", name="osd_scrub_min_interval")
+        mon_obj.remove_config(section="osd", name="osd_scrub_max_interval")
+        mon_obj.remove_config(section="osd", name="osd_deep_scrub_interval")
+
+        if not check_for_scrub(rados_obj, wait_time):
+            log.error(
+                "The scrub operations are still in progress.Not executing the further tests"
+            )
+            return 1
+
         if not rados_obj.create_erasure_pool(name=pool_name, **ec_config):
             log.error("Failed to create the EC Pool")
             return 1
+        # Get the acting PG set
+        acting_pg_set = rados_obj.get_pg_acting_set(pool_name=pool_name)
 
         if config.get("debug_enable"):
             mon_obj.set_config(section="osd", name="debug_osd", value="20/20")
             mon_obj.set_config(section="mgr", name="debug_mgr", value="20/20")
         method_should_succeed(wait_for_clean_pg_sets, rados_obj)
+
         pg_info = rados_obj.create_ecpool_inconsistent_obj(
             objectstore_obj, client_node, pool_name, req_no_of_objects
         )
+        scrub_object.set_osd_flags("set", "nodeep-scrub")
+        scrub_object.set_osd_flags("set", "noscrub")
+
+        pg_id, no_of_inconsistent_objects = pg_info
```

Comment:
This can cause a exception to be raised if the pg_info is None. remove the check